int32 from_id
bool[] occupancy
geometry_msgs/Point[] positions